avocado/plugins/runner_nrunner.py

```diff
@@ -169,6 +171,25 @@ class Runner(RunnerInterface):
     name = 'nrunner'
     description = 'nrunner based implementation of job compliant runner'
 
+    @staticmethod
+    def _update_avocado_configuration_used_on_runnables(runnables, config):
+        """Checks if runnables have runner requirements fulfilled
```

Comment:
IMO this description doesn't match the functionality. The method updates missing configuration, doesn't check if runnable have runner requirements fulfilled.

Reply:
It certainly doesn't! It was probably a copy/paste mistake.  Thanks for spotting it.